hw/ip/adc_ctrl/data/adc_ctrl_sec_cm_testplan.hjson

```diff
@@ -0,0 +1,27 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// Security countermeasures testplan extracted from the IP Hjson using reggen.
+//
+// This testplan is auto-generated only the first time it is created. This is
+// because this testplan needs to be hand-editable. It is possible that these
+// testpoints can go out of date if the spec is updated with new
+// countermeasures. When `reggen` is invoked when this testplan already exists,
+// It checks if the list of testpoints is up-to-date and enforces the user to
+// make further manual updates.
+{
+  testpoints: [
+    {
+      name: sec_cm_bus_integrity
+      desc: '''
+            Verifies the countermeasure BUS.INTEGRITY.
+
+            Countermeasure description:
+            End-to-end bus integrity scheme.
```

Comment:
i think the `desc` is meant to describe what to test and how. Replicating the design description here seems a bit redundant and may bring more work for maintenance. 

If we want to show both design description and test description, we can show them in the rendered document. For hjson, we'd better put the necessary info. For example, we could put the testpoint along with countermeasures, and describe the detail of the testpoint in testplan. In rendered DV DOC, we can show countermeasure description, testpoint and testpoint description all together.

```
  countermeasures: [
    { name: "BUS.INTEGRITY",
      desc: "End-to-end bus integrity scheme."
      testpoint: "tl_intg_err"
    }

```